sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -67,64 +64,63 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
   private abortControllers: Map<string, AbortController[] | undefined>;
 
   constructor(options?: TokenCredentialOptions) {
-    super(
-      undefined,
-      createPipelineFromOptions({
-        ...options,
-        deserializationOptions: {
-          expectedContentTypes: {
-            json: ["application/json", "text/json", "text/plain"]
-          }
-        }
-      })
-    );
-
-    this.baseUri = this.authorityHost = getIdentityClientAuthorityHost(options);
+    const packageDetails = `azsdk-js-identity/2.0.0-beta.5`;
```

Comment:
Add this variable to the `constantsPath` list in `package.json` similar to https://github.com/Azure/azure-sdk-for-js/blob/8d095984fc23900dc2206ec7511cd20ff8ccd80e/sdk/textanalytics/ai-text-analytics/package.json#L33-L36

Reply:
oh cool!

Reply:
Done! https://github.com/Azure/azure-sdk-for-js/pull/16514/commits/37e33ef42274102585756bd76b55e4326bd5216c